rules/rules.libsonnet

```diff
@@ -243,102 +182,6 @@
-            expr: |||
-              (node:node_memory_bytes_total:sum - node:node_memory_bytes_available:sum)
-              /
-              node:node_memory_bytes_total:sum
-            |||,
-          },
-          {
-            // Memory utilisation per node, normalized by cluster-wide memory
-            record: 'node:cluster_memory_utilisation:ratio',
-            expr: |||
-              (node:node_memory_bytes_total:sum - node:node_memory_bytes_available:sum)
-              /
-              scalar(sum(node:node_memory_bytes_total:sum))
-            |||,
-          },
-          {
-            record: ':node_memory_swap_io_bytes:sum_rate',
-            expr: |||
-              1e3 * sum(
-                (rate(node_vmstat_pgpgin{%(nodeExporterSelector)s}[1m])
-               + rate(node_vmstat_pgpgout{%(nodeExporterSelector)s}[1m]))
-              )
-            ||| % $._config,
-          },
-          {
-            // DEPRECATED
-            record: 'node:node_memory_utilisation:',
-            expr: |||
-              1 -
-              sum by (node) (
-                (node_memory_MemFree_bytes{%(nodeExporterSelector)s} + node_memory_Cached_bytes{%(nodeExporterSelector)s} + node_memory_Buffers_bytes{%(nodeExporterSelector)s})
-              * on (namespace, %(podLabel)s) group_left(node)
-                node_namespace_pod:kube_pod_info:
-              )
-              /
-              sum by (node) (
-                node_memory_MemTotal_bytes{%(nodeExporterSelector)s}
-              * on (namespace, %(podLabel)s) group_left(node)
-                node_namespace_pod:kube_pod_info:
-              )
-            ||| % $._config,
-          },
-          {
-            // DEPENDS 2018-02-08
-            // REPLACE node:node_memory_utilisation:
-            record: 'node:node_memory_utilisation_2:',
-            expr: |||
-              1 - (node:node_memory_bytes_available:sum / node:node_memory_bytes_total:sum)
-            ||| % $._config,
-          },
-          {
-            record: 'node:node_memory_swap_io_bytes:sum_rate',
-            expr: |||
-              1e3 * sum by (node) (
-                (rate(node_vmstat_pgpgin{%(nodeExporterSelector)s}[1m])
-               + rate(node_vmstat_pgpgout{%(nodeExporterSelector)s}[1m]))
-               * on (namespace, %(podLabel)s) group_left(node)
-                 node_namespace_pod:kube_pod_info:
-              )
-            ||| % $._config,
-          },
-          {
-            // Disk utilisation (seconds spent, by rate() it's bound by 1 second)
-            record: ':node_disk_utilisation:avg_irate',
-            expr: |||
-              avg(irate(node_disk_io_time_seconds_total{%(nodeExporterSelector)s,%(diskDeviceSelector)s}[1m]))
-            ||| % $._config,
-          },
           {
             // Disk utilisation (seconds spent, by rate() it's bound by 1 second)
             record: 'node:node_disk_utilisation:avg_irate',
```

Comment:
Perhaps I missed it in my list, but this one can also go, IMHO.

Reply:
removed